metadata-models/src/main/pegasus/com/linkedin/schema/SchemaField.pdl

```diff
@@ -84,6 +84,11 @@ record SchemaField {
    */
   isPartOfKey: boolean = false
 
+  /**
+   * For Datasets which are partitioned, this determines the partitioning key.
+   */
+  isPartitioningKey: optional boolean
```

Comment:
this will support partitions on multiple columns, right? Do we need to supply an ordering of the columns as well, or is just marking the partition key columns sufficient?

Reply:
It's a good question. If its hierarchical we likely would need another field to mark the "order" or "index" within the key